zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -440,6 +440,7 @@ pub fn shield_transparent_funds<E, N, P, D, R, U>(
     params: &P,
     prover: impl TxProver,
     usk: &UnifiedSpendingKey,
```

Comment:
Presumably when Orchard is supported, this is going to shield to the most preferred pool in the USK, which will prefer Orchard over Sapling. Should that be specified now in the doc comment, rather than saying it shields to the default Sapling address?

Reply:
Noted in https://github.com/zcash/librustzcash/issues/490